blockchain/v0/reactor.go

```diff
@@ -358,7 +358,7 @@ FOR_LOOP:
 			// NOTE: we can probably make this more efficient, but note that calling
 			// first.Hash() doesn't verify the tx contents, so MakePartSet() is
 			// currently necessary.
-			err := state.Validators.VerifyCommit(
+			err := state.Validators.VerifyCommitLight(
```

Comment:
I think this was called `VerifyCommitLight` specifically to make it clear that it was only used in the context of a light client, since we had a security issue where it was used for something else where it wasn't appropriate. We may want to add a small `VerifyCommitFast()` shim or something. I'll defer to @melekes.

Reply:
I can make this change. Light could also mean lighter verification, not full verification 😛

Reply:
I think it's fine to use `VerifyCommitLight` here